GameOfLife.java

```diff
@@ -85,22 +114,50 @@ public static int[][] evolve(int[][] board) {
 	// Assumes that j is at least 1 and at most the number of columns in the board - 1. 
 	// Uses the count(board,i,j) function to count the number of alive neighbors.
 	public static int cellValue(int[][] board, int i, int j) {
-		//// Replace the following statement with your code.
-		return 0;
+		int value = 0;
+		if(board[i][j] == 1){
+			if(count(board, i, j) < 2 || count(board, i, j) > 3){
+				value = 0;
+			}
+			else if(count(board, i, j) == 2 || count(board, i, j) == 3){
```

Comment:
`else` is enough here, the `if` doesn't specify more than included in that case.
```suggestion
			else {
```